### PR TITLE
[usecase-embeddingapi]Add test for set/getSaveFormData methods

### PR DESCRIPTION
**Pull request by haoxli**
Impacted tests(approved): new 2, update 0, delete 0
Unit test platform: Crosswalk for Android 21.51.546.0
Unit test result summary: pass 2, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-1821

### CODE REVIEW

**Comment by haoxli** [comment]
@Shao-Feng PTAL

**Comment by haoxli** [comment]
@Shao-Feng  updated README.

File: usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingSaveFormDataAsync.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                .append("4. Input first letter of your name or double click input area.\n\n")
+                .append("5. Set SaveFormData to false.\n\n")
+                .append("6. Input first letter of your name again or double click input area.\n\n")
+                .append("Expected Result:\n\n")
+                .append("Test passes if the saved form date does not auto complete(or popup) after set SaveFormData to false.");
+        new  AlertDialog.Builder(this)
+                .setTitle("Info")
+                .setMessage(mess.toString())
+                .setPositiveButton("confirm", null)
+                .show();
```
Comment (Shao-Feng) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingSaveFormDataAsync.java]: Besides the popup, the test step should be shown in the main activity also as what we did in other use case

Comment (haoxli) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingSaveFormDataAsync.java]: Added test steps in layout xml.

**Pull request merged**